utils/parse_config.py

```diff
@@ -64,22 +64,35 @@ def generate_gear_args(gear_context, FLAG):
     elif FLAG == "dcm2niix":
 
         # Notice the explicit 'y' for bids_sidecar, in order to capture metadata; the
-        # user-defined config option setting wil be considered during gear resolve.
+        # user-defined config option setting wil be considered during the gear resolve stage.
+        filename = gear_context.config["filename"]
+        filename = filename.replace(" ", "_")
```

Comment:
The dcm2niix tool will catch errors related to the filename. And errors that impact the output files in Flywheel would then be caught on Gear completion. I think it would be good to have the sanitization of output filenames, but I am not sure all that that entails and is really more of a Gear construction issue than a specific dcm2niix issue - or rather something for the Gear Toolkit. Something to bring to the team at least!